### ISSUE TITLE
Headers blocking division starting at fridays

### ISSUE THREAD

**Issue by labs20** [ANovokmet/svelte-gantt#123]
Hi. Thanks for your work.

I'm trying to set up a gantt with this settings:
```
new SvelteGantt({
                target: this.$refs.gantt.$el,
                props: {
                    dateAdapter: new MomentSvelteGanttDateAdapter(moment),
                    from: this.inicio,
                    to: this.final,

                    rows: rows,
                    tasks: tasks,
                    dependences: dependences,

                    reflectOnParentRows: false,
                    reflectOnChildRows: false,

                    highlightWeekends: true,
                    highlightColor: 'rgba(239,69,69,0.18)',

                    zoomLevels: [],

                    columnUnit: 'day',
                    columnOffset: 1,

                    magnetUnit: 'day',
                    magnetOffset: 1,

                    rowHeight: 28,
                    rowPadding: 2,

                    fitWidth: true,
                    minWidth: 2000,

                    tableWidth: 240,
                    tableHeaders: [
                        { title: 'Tarefas', property: 'label', width: 140, type: 'tree',  },
                       // { title: 'Duração', property: 'label', width: 100, type: 'tree',  },
                    ],
                    headers: [{ unit: 'day', format: 'MMMM YYYY', offset: 7 }, { unit: 'day', format: '  DD  ddd ' }],

                    ganttTableModules: [SvelteGanttTable],
                    ganttBodyModules: [SvelteGanttDependencies]
                }
            });
```

The first header is dividing the days in blocks of seven all right, but it is starting by friday, and not by sunday or monday. I have configured the header with `unit: 'week'` also, but still the same problem.

<img width="1133" alt="Captura de Tela 2022-11-30 às 18 03 48" src="https://user-images.githubusercontent.com/4047789/204907636-730e646c-dcd9-4e17-b1ca-65dc3f8f9661.png">
 
20 qui => thursday
21 sex => friday
22 sáb => saturday
23 dom => sunday

Can you please help me to find what I'm doing wrong here?

Thank you!

**Comment by labs20** [comment]
Mmmmm... Looks like the header's grouping start point is the day you set to `from` prop, in disregard of some logical day of the week. Odd.

I "fix it" with this workaround `from: this.inicio.day(0);`. Assuming that `this.inicio` is a moment date, `day(0)` sets it to the sunday of the date's week.

Hope it helps.

**Comment by V-Py** [comment]
I think this might be fixed since #136
